packages/clerk-js/src/ui/components/UserProfile/__tests__/ProfilePage.test.tsx

```diff
@@ -161,7 +159,7 @@ describe('ProfilePage', () => {
       expect(fixtures.clerk.user?.setProfileImage).toHaveBeenCalledWith({ file: null });
     });
 
-    xit('"Remove image" is not shown when a default image exists', async () => {
+    it.skip('"Remove image" is not shown when a default image exists', async () => {
```

Comment:
Are we skipping on purpose ?

Reply:
it's already skipped. This change just replaces the `xit` alias to the `it.skip`. It's the same.